common/src/main/java/dev/ftb/mods/ftbquests/client/gui/quests/QuestScreen.java

```diff
@@ -406,6 +409,63 @@ public boolean keyPressed(Key key) {
 					return moveSelectedQuests(step, 0D);
 				}
 			}
+			switch (key.keyCode) {
+				case GLFW.GLFW_KEY_C -> {
+					Collection<Quest> quests = getSelectedQuests();
```

Comment:
Would like to see this split out into its own method for tidiness.  It's too much code for a switch case.

Reply:
yup, I'll refactor into separate methods to use with right click + copy or paste